app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+require('dotenv').config({ path: './config/test.env' }); // Default: path.resolve(process.cwd(), '.env')
+
+const messagesRoute = require('./api/messages/messages'); // Access to to this file 
```

Comment:
Create v1 direcotry under api (/api/v1).
under this directory add index.js file to import al export all the routes.